pages/docs/docs.md

```diff
@@ -1543,35 +1543,8 @@ Different endpoints can have different rate limits. It works as follows:
 ---
 
 ### Android
-To use Javalin in an Android project, you will need to:
 
-*1: Target the Android SDK 26 and higher:*
-
-```
-defaultconfig {
-  minSdkVersion 26
-  targetSdkVersion 28
-}
-```
-
-*2: Target Java 8:*
-
-```
-compileOptions {
-    sourceCompatibility JavaVersion.VERSION_1_8
-    targetCompatibility JavaVersion.VERSION_1_8
-}
-```
-
-*3: Insert this in your build.gradle file:*
-
-```
-packagingOptions {
-  exclude 'org/eclipse/jetty/http/encoding.properties'
-}
-```
-
-*4: Specify `android.enableD8=true` in your `gradle.properties` file.*
+Due to some issues out of our control, Javalin 5 is not compatible with Android, but Javalin 4 is. You can find the docs for Javalin 4 [here](/archive/docs/v4.6.X.html).
```

Comment:
This seems a bit unnecessarily vague, maybe we should explain and link to an issue?

Reply:
I thought about it, but does the user really care? I can add it for sure.

Reply:
I think a lot of people would want to know why and if it's eventually coming, so they can decide if it's worth to even use Javalin 4 🤔

Reply:
What about 

> "Due to [Jetty 11 being incompatible with Android](https://github.com/javalin/javalin.github.io/issues/211#issuecomment-1438319603), Javalin 5 is not compatible either, but Javalin 4 is. You can find the docs for Javalin 4 [here](/archive/docs/v4.6.X.html)."

Reply:
Is it going to be fixed in jetty11? Or in jetty 12? Ever? This is something we should answer before pointing people to javalin 4. I would rather point people to a non-jetty project than javalin 4, which uses end of lifed jetty 9.

Reply:
> Is it going to be fixed in jetty11? Or in jetty 12? Ever? This is something we should answer before pointing people to javalin 4. I would rather point people to a non-jetty project than javalin 4, which uses end of lifed jetty 9.

I don't really think so:
https://github.com/javalin/javalin.github.io/issues/211#issuecomment-1438319603

I'll look up some alternative designed or at least tested on Android, if there are.

Reply:
I'm a bit confused. Based on the thread over at the Jetty tracker, it works for Android 13, so why aren't we saying that, rather than that it doesn't work?

Reply:
> I'm a bit confused. Based on the thread over at the Jetty tracker, it works for Android 13, so why aren't we saying that, rather than that it doesn't work?

It does say that, however it does not.

There might be pieces of Jetty that work, but Javalin definitely doesn't. I tried it myself and asked a colleague to run test himself independently and we both got the same result. Both using the latest SDK for Android 13 and Java 11, still no luck.

Android is a big mess when it comes to Java apis and it really shows here when using high level newer APIs. 

I can continue the issue you mentioned, but it's a lot of work for them and I doubt they'll have android as a priority for development.

Reply:
> I can continue the issue you mentioned, but it's a lot of work for them and I doubt they'll have android as a priority for development.

We should at least tell them, so they can make an informed decision on the matter 😊

Reply:
[Done](https://github.com/eclipse/jetty.project/issues/8912#issuecomment-1439716937)

Reply:
Thanks! They just reopened it, so seems they are interested in making it work. I'm okay with pointing to 4x then. Could you add a link to the Jetty issue too (in this PR)?